services/workflows-service/src/customer/api-key/api-key.repository.ts

```diff
@@ -0,0 +1,43 @@
+import { Injectable } from '@nestjs/common';
+import { PrismaService } from '@/prisma/prisma.service';
+import { ApiKey, Prisma } from '@prisma/client';
+
+@Injectable()
+export class ApiKeyRepository {
+  constructor(protected readonly prisma: PrismaService) {}
+
+  async create<T extends Prisma.ApiKeyCreateArgs>(
+    args: Prisma.SelectSubset<T, Prisma.ApiKeyCreateArgs>,
+  ): Promise<ApiKey> {
+    return await this.prisma.apiKey.create<T>(args);
+  }
+
+  // eslint-disable-next-line ballerine/verify-repository-project-scoped
+  async find(hashedKey: string) {
```

Comment:
you can change the function names to `findUnscoped` and `deleteUnscoped` and you won't need the `eslint-disable-next-line` comments